styleguide/components/01-mixins/themify/_themify.scss

```diff
@@ -183,14 +183,9 @@ $themes: map-merge($themes, $overwrite-themes);
   @each $theme, $colors in $themes {
     // Apply the default section as default color scheme if there is no section
     // defined in the DOM.
-    @if $theme == "default" {
+    .cs--#{$theme} & {
```

Comment:
een body zonder een color scheme heeft dug geen gethemede atoms?

Reply:
At this moment, no. Because this gives an error with specificity...  :/

Reply:
okey, let's add a class to the body element in our preview template :-D